lib/gis/file_name.c

```diff
@@ -27,8 +27,29 @@ static void append_char(char*, char);
   If <i>name</i> is of the form "nnn@ppp" then path is set as if name
   had been "nnn" and mapset had been "ppp" (mapset parameter itself is
   ignored in this case).
-  
-  \param[out] path buffer to hold resultant full path to file
+
+  Paths to files are in form:
+  /home/user/GISDBASE/LOCATION/mapset/element/name
+
+  path input buffer memory must be preallocated by caller.
+
+  C:
+  @code
+  char path[GPATH_MAX];
+  @endcode
+  Python:
+  @code
```

Comment:
You can tell Doxgen this is Python: https://www.doxygen.nl/manual/commands.html#cmdcode

Or you can use just Markdown syntax with 3 backticks followed by simple `py` (although the doc for that is less specific).

Reply:
I added code hints to both C an Py parts. Doxygen was already doing a good job with highlighting though.